cmd/neofs-cli/modules/storagegroup/put.go

```diff
@@ -43,10 +43,19 @@ func initSGPutCmd() {
 	_ = sgPutCmd.MarkFlagRequired(sgMembersFlag)
 
 	flags.Uint64(commonflags.Lifetime, 0, "Storage group lifetime in epochs")
-	_ = sgPutCmd.MarkFlagRequired(commonflags.Lifetime)
+	flags.Uint64P(commonflags.ExpireAt, "e", 0, "The last active epoch of the storage group")
+	sgPutCmd.MarkFlagsMutuallyExclusive(commonflags.ExpireAt, commonflags.Lifetime)
+	_ = flags.MarkDeprecated("lifetime", "use --expire-at flag instead for setting last epoch of the storage group")
 }
 
 func putSG(cmd *cobra.Command, _ []string) {
+	//with next release of cobra we can use this instead of below
```

Comment:
what exact release is required?

Reply:
which will consist https://github.com/spf13/cobra/blob/fd865a44e3c48afeb6a6dbddadb8a5519173e029/flag_groups.go#L49 (probably 1.7.1)